content/shmem_quiet.tex

```diff
@@ -38,23 +34,21 @@
 
 \apinotes{
     \FUNC{shmem\_quiet} is most useful as a way of ensuring completion of
-    several memory store, blocking \PUT{}, \ac{AMO}, and \OPR{put-with-signal},
-    as well as nonblocking \PUT{},
-    \OPR{put-with-signal}, \GET{}, and \ac{AMO} routines to symmetric data objects initiated by the calling
-    \ac{PE}.  For example, one might use \FUNC{shmem\_quiet} to await delivery
+    several operations to symmetric data objects initiated by the calling
+    \ac{PE}. For example, one might use \FUNC{shmem\_quiet} to await delivery
     of a block of data before issuing another \PUT{} or nonblocking
     \PUT{} routine, which sets a completion flag on another \ac{PE}.
-     \FUNC{shmem\_quiet} is not usually needed if
+    \FUNC{shmem\_quiet} is not usually needed if
     \FUNC{shmem\_barrier\_all} or \FUNC{shmem\_barrier} are called.  The barrier
-    routines wait for the completion of outstanding writes (memory store,
-    blocking \PUT{}, \ac{AMO}, and \OPR{put-with-signal}, as well as nonblocking \PUT{},
-    \OPR{put-with-signal}, \GET{}, and \ac{AMO} routines) to
+    routines wait for the completion of outstanding writes to
     symmetric data objects on all \acp{PE}.
 
     In an \openshmem program with multithreaded \acp{PE}, it is the
-    user's responsibility to ensure ordering between operations issued by the threads
-    in a \ac{PE} that target symmetric memory (e.g. \PUT{}, \ac{AMO}, \OPR{put-with-signal}, memory stores,
-    and nonblocking routines) and calls by threads in that \ac{PE} to
+    user's responsibility to ensure ordering between operations issued by the
+    threads in a \ac{PE} that target symmetric memory
+    (e.g. \PUT{}, \ac{AMO}, \OPR{put-with-signal}, memory stores,
+    and nonblocking routines)
```

Comment:
What's the reason for why this list of operations in parentheses were left in when the others were removed?